win32ss/gdi/ntgdi/freetype.c

```diff
@@ -334,6 +334,122 @@ SharedFace_Release(PSHARED_FACE Ptr)
     IntUnLockFreeType();
 }
 
+VOID DumpFontGDI(PFONTGDI FontGDI)
```

Comment:
In case all of these functions are used *only* inside `#ifndef NDEBUG` blocks, I would suggest to surround all of these inside a single `#if DBG ... #endif` block.

Reply:
@learn-more How about @HBelusca's suggestion?

Reply:
Not referenced functions will be automatically ignored by compilers. These functions can be used in the other places for debugging.

Reply:
OK.